eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxos.hs

```diff
@@ -82,42 +88,50 @@ import NoThunks.Class (NoThunks)
 -- The UTXOS transition system
 --------------------------------------------------------------------------------
 
+{- These are the Field constraints needed. Since this STS instance is only going to
+   be used once, We can make all these go away by fixing the Core.XX types to the
+   concrete types used in this instance.
+type UtxosFieldNeeds era =
```

Comment:
can we delete this comment?

Reply:
sure. That was really just notes for when comparing the old and new versions